google-ads-examples/src/main/java/com/google/ads/googleads/examples/errorhandling/HandleRateExceededError.java

```diff
@@ -207,7 +207,9 @@ public void run() {
               MutateAdGroupCriteriaResponse response =
                   adGroupCriterionServiceClient.mutateAdGroupCriteria(
                       Long.toString(customerId), operations, false, true);
-
+              // The result set will be empty while validateOnly is set to true in the
+              // mutateAdGroupCriteria request. Remove the validateOnly parameter or set to false
+              // to complete the request and return a result set.
```

Comment:
Does it make sense to do 
System.Out.PrintLn("%d operations validated.");

// The result set will be empty while validateOnly is set to true in the
// mutateAdGroupCriteria request. Remove the validateOnly parameter to false
// and uncomment the code below to complete the request and return a result set.
System.out.printf("Added %d ad group criteria:%n", response.getResultsCount());
for (MutateAdGroupCriterionResult result : response.getResultsList()) {
  System.out.println(result.getResourceName());

Reply:
Yes, done.